package.json

```diff
@@ -11,20 +11,17 @@
     "build:dist": "ethereumjs-config-build",
     "prepublishOnly": "npm run lint && npm run build:dist && npm run testBuildIntegrity",
     "coverage": "nyc npm run coverageTests && nyc report --reporter=text-lcov > .nyc_output/lcov.info",
-    "coverageTests": "npm run build:dist && tape './tests/api/**/*.js' ./tests/tester.js -s --dist",
+    "coverageTests": "npm run build:dist && tape './tests/api/**/*.js' ./tests/tester.js --state --dist",
     "coveralls": "npm run coverage && if [ -n \"$COVERALLS_REPO_TOKEN\" ]; then coveralls <.nyc_output/lcov.info; fi",
     "docs:build": "typedoc lib",
-    "testVM": "node ./tests/tester -v",
-    "testStateByzantium": "npm run build:dist && node ./tests/tester -s --fork='Byzantium' --dist",
-    "testStateConstantinople": "npm run build:dist && node ./tests/tester -s --fork='Constantinople' --dist",
-    "testStatePetersburg": "npm run build:dist && node ./tests/tester -s --fork='Petersburg' --dist",
-    "testStateIstanbul": "npm run build:dist && node ./tests/tester -s --fork='Istanbul' --dist",
-    "testBuildIntegrity": "npm run build:dist && node ./tests/tester -s --dist --test='stackOverflow'",
-    "testBlockchain": "npm run build:dist && node --stack-size=1500 ./tests/tester -b --fork='Istanbul' --dist --excludeDir='GeneralStateTests'",
-    "testBlockchainPetersburg": "npm run build:dist && node --stack-size=1500 ./tests/tester -b --fork='Petersburg' --dist --excludeDir='GeneralStateTests'",
-    "testBlockchainGeneralStateTests": "npm run build:dist && node --stack-size=1500 ./tests/tester -b --dist --dir='GeneralStateTests'",
-    "testAPI": "npm run build:dist && tape './tests/api/**/*.js'",
-    "testAPI:browser": "npm run build:dist && karma start karma.conf.js",
+    "test:vm": "node ./tests/tester --vm",
```

Comment:
Just a note: this test script has [been disabled](https://github.com/ethereumjs/ethereumjs-vm/blob/master/tests/tester.js#L60)

Reply:
Yes, this should rather be completely removed, this is again and again some source for confusion for people (and right so).

Reply:
What was the original purpose of the VM tests? If they provide no benefit anymore then let’s definitely remove them, but if they would be valuable and could be fixed given some attention, I can allocate some time toward it in the future.